HW_3/task16.py

```diff
@@ -0,0 +1,18 @@
+#Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A. 
+#Пользователь в первой строке вводит натуральное число N – количество элементов в массиве.
+#В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). Последняя строка содержит число X.
+
+import random
+print('Введите число элментов массива ')
+n = int(input())
+print('Введите искомое число ')
+x = int(input())
```

Comment:
Почему вы во сих пор используете связку `print-input` таким образом, когда можно делать так:
```suggestion
n = int(input('Введите число элементов массива '))
x = int(input('Введите искомое число '))
```